processing/data_collection/gazette/spiders/pr_cascavel.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+from dateparser import parse
+import datetime as dt
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class PrCascavelSpider(BaseGazetteSpider):
+    TERRITORY_ID = '4104808'
+    name = 'pr_cascavel'
+    allowed_domains = ['cascavel.pr.gov.br']
+    start_urls = ['http://www.cascavel.pr.gov.br/servicos/orgao_oficial.php']
+    download_url = 'http://www.cascavel.pr.gov.br/anexos/{}'
+
+    def parse(self, response):
+        """
+        @url http://www.cascavel.pr.gov.br/servicos/orgao_oficial.php
+        @returns items 1
+        @scrapes date file_urls is_extra_edition territory_id power scraped_at
+        """
+        start_date = dt.date(2015, 1, 1)
+        parsing_date = dt.date.today()
+        for row in response.css('table tr')[1:]:
+            cols = row.css('td')
+            edition = cols[0].css('font::text').extract()
+            date = cols[1].css('font::text').extract_first()
+            date = parse(date, languages=['pt']).date()
+            if date < start_date:
+                break
+            for link in cols[2].css('a'):
+                link_text = link.css('::text').extract_first()
+                power = 'executive' if 'Executivo' in link_text else 'legislature'
+                url = link.css('::attr(href)').extract_first()[10:]
```

Comment:
Quoting @rennerocha here: 

> I believe that you got a relative URL. So use response.urljoin to generate the absolute URL instead of doing string manipulation (https://doc.scrapy.org/en/latest/topics/request-response.html#scrapy.http.Response.urljoin).